app/views/homepage/index.html.erb

```diff
@@ -128,30 +128,30 @@
         </div>
 
         <div class="govuk-grid-column-one-third">
-          <a href="/government/topical-events/budget-2020" aria-hidden="true" tabindex="-1">
-            <%= image_tag "homepage/budget-2020--480w.jpg", {
+          <a href="/government/topical-events/a-plan-for-jobs-2020" aria-hidden="true" tabindex="-1">
+            <%= image_tag "homepage/plan-for-jobs-2020--480w.png", {
               alt: "",
               class: "home-promo__image",
               height: 400,
               loading: "lazy",
               sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
               srcset: {
```

Comment:
! Very cool, i'd heard about srcset but not see it used.
Nice

Reply:
It was @injms who laid the groundwork for this!

Reply:
@injms legend.